alb/main.tf

```diff
@@ -1,16 +1,26 @@
 # Create a new load balancer
 resource "aws_lb" "alb" {
   load_balancer_type         = "application"
-  name_prefix                = "${var.name_prefix}"
-  internal                   = "${var.internal}"
-  subnets                    = ["${var.subnets}"]
-  security_groups            = ["${aws_security_group.sg_alb.id}"]
-  enable_deletion_protection = "${var.enable_deletion_protection}"
-  access_logs                = ["${var.access_logs}"]
+  name                       = "${var.project}-${var.environment}-${var.name_prefix}"
```

Comment:
This will cause breaking changes at the AWS level and should be carefully thought out. So imo it should not be part of this PR and should be discussed as part of a more general naming strategy to apply to all our modules.